nav2_behavior_tree/plugins/action/drive_on_heading_action.cpp

```diff
@@ -37,13 +37,16 @@ void DriveOnHeadingAction::initialize()
   getInput("speed", speed);
   double time_allowance;
   getInput("time_allowance", time_allowance);
+  double disable_collision_checks;
```

Comment:
This should be a bool

Reply:
Good catch. I'm Surprised the compiler didn't complain.